NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Things Paul Came Across
+
+## General Python housekeeping I could do later
+
+- *Proper package*. Make imports more normal by changing `t-html` to something legally importable.
```

Comment:
agreed, the name is made up and I am open to any suggestion

Reply:
Ok, I'll propose something in the PR.